package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "percy-seleniumjs",
+  "version": "0.1.0-alpha.1",
+  "description": "Unofficial SeleniumJS SDK for Percy. Works with Protractor too.",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rimraf lib/* && babel src --out-dir lib --delete-dir-on-start",
```

Comment:
noooooooo babel

Reply:
can I get away without babel? maybe if I just don't use the import/export syntax? what about async/await?

Reply:
async await is fine in newest node

Reply:
you can use everything in node 10 except import/export. async/await has been supported since like node 6

Reply:
plus node 10 just hit LTS so it's fine to limit for OSS stuff i think

Reply:
Great, amazing. Will do that.